packages/renderer/src/lib/agent-workspaces/AgentWorkspaceDetailsOverview.svelte

```
@@ -9,55 +9,63 @@ import {
+  workspaceSummary
+    ? $agentInfos.find(a => {
+        console.log('matching "', agentDef, '" against', a.id);
+        return a.id === agentDef;
+      })
+    : undefined,
+);
 
 const resolvedModel = $derived(
-  workspaceSummary?.model ? $catalogModels.find(m => getModelId(m) === workspaceSummary.model) : undefined,
+  // eslint-disable-next-line no-constant-condition
+  false ? $catalogModels.find(m => getModelId(m) === '') : undefined,
```

Biome (2.5.0)

[error] This condition always evaluates to the same value.

(lint/correctness/noConstantCondition)